src/data-generator/declare.ts

```diff
@@ -342,6 +349,14 @@ export async function createDeathDeclaration(
       contact: 'SPOUSE',
       contactPhoneNumber:
         '+2607' + faker.datatype.number({ min: 10000000, max: 99999999 }),
+      attachments: Array.from({ length: NUMBER_OF_ATTACHMENTS_PER_RECORD }).map(
+        () => ({
+          contentType: 'image/png',
+          data: 'data:image/png;base64,' + ATTACHMENT,
+          subject: AttachmentSubject.DeceasedDeathCauseProof,
+          type: AttachmentType.CoronersReport
+        })
+      ),
```

Comment:
I wonder if having the same types of attachments, duplicated would affect the record

Reply:
Good catch, guess it would be better if they were different. Maybe something like

```ts
const DEATH_ATTACHMENT_TYPES = [
  AttachmentType.AttestedLetterOfDeath,
  AttachmentType.BirthCertificate,
  AttachmentType.BurialReceipt,
 //  .. etc
]

// ...

 attachments: Array.from({ length: NUMBER_OF_ATTACHMENTS_PER_RECORD }).map(
   (_, i) => ({
     // ...,
     type: DEATH_ATTACHMENT_TYPES[i % NUMBER_OF_ATTACHMENTS_PER_RECORD]
   })
 )

```
do we have that kind of array already generated by gateway types or does core have hard coded array for this we could think about reusing?

Reply:
I'll include this fix in the form-config-to-country branch